src/pages/Examples.spec.tsx

```diff
@@ -1,16 +1,94 @@
+import Block from '@dojo/framework/widget-core/meta/Block';
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
 import harness from '@dojo/framework/testing/harness';
+import { Constructor, WidgetMetaConstructor, MetaBase } from '@dojo/framework/widget-core/interfaces';
 import { tsx } from '@dojo/framework/widget-core/tsx';
+import { w } from '@dojo/framework/widget-core/d';
+
+import FontAwesomeIcon from '../widgets/icon/FontAwesomeIcon';
+import Grid from '../widgets/grid/Grid';
+import Landing from '../widgets/Landing';
+import LinkedCard from '../widgets/card/LinkedCard';
+import Subsection from '../widgets/section/Subsection';
 
 import Examples from './Examples';
 import * as css from './Examples.m.css';
 
+const mockMetaMixin = <T extends Constructor<WidgetBase<any>>>(Base: T, mockStub: jest.Mock): T => {
+	return class extends Base {
+		protected meta<T extends MetaBase>(MetaType: WidgetMetaConstructor<T>): T {
+			return mockStub(MetaType);
+		}
+	};
+};
+
+const mockExamplesBlock = jest.fn();
+
+const mockBlockRun = jest.fn().mockImplementation((input: any) => {
+	return mockExamplesBlock;
+});
+
+const mockMeta = jest.fn().mockImplementation((input: any) => {
+	if (Block) {
+		return {
+			run: mockBlockRun
+		};
+	}
+});
+
 describe('Examples', () => {
 	it('renders', () => {
-		const h = harness(() => <Examples />);
+		mockExamplesBlock.mockReturnValueOnce([
+			{
+				code: 'code',
+				demo: 'demo',
+				example: 'example',
+				exampleName: 'name',
+				overview: 'overview',
+				sandbox: true
+			}
+		]);
+		const h = harness(() => w(mockMetaMixin(Examples, mockMeta), {}));
```

Comment:
Should be able to use `tsx` if you create do something like 
```ts
const ExampleWithMock = mockMetaMixin(Examples, mockMeta);
``` 
before using it 
```ts
const h = harness(() => <ExampleWithMock />);
```

(you don't have to use my rubbish name though!)